code/__DEFINES/diseases.dm

```diff
@@ -8,6 +8,7 @@
 
 //Disease Flags
 #define CURABLE (1<<0)
+#define CHRONIC (1<<0)
```

Comment:
Curable and Chronic appear to be using the same value, I'm pretty sure this is going to cause them to be treated as the same when used later. You probably want to give chronic a new value. (1<<3) should be fine probably.